example/src/main/tut/typeclass/Semigroup.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title:  "Semigroup"
+---
+
+# Semigroup
+
+**Category Theory**
+
+*A semigroup is an associative magma. It contains a set and an associative binary operation.*
```

Comment:
Mentioning magmas seems a little strange without a definition. A link to wiki/ncatlab may be welcome.

I'm not sure if it "contains" them; maybe
> ... magma. That is, it is a set along with an associative binary operation.

Reply:
Unless we have magma in scalaz, I would avoid mentioning it altogether.